db/sql/test_00_msar.sql

```diff
@@ -1310,47 +1310,40 @@ END;
 $$ LANGUAGE plpgsql;
 
 
-CREATE OR REPLACE FUNCTION __setup_alter_schema() RETURNS SETOF TEXT AS $$
+CREATE OR REPLACE FUNCTION test_patch_schema() RETURNS SETOF TEXT AS $$
+DECLARE sch_oid oid;
 BEGIN
-  CREATE SCHEMA alter_me;
-END;
-$$ LANGUAGE plpgsql;
-
+  CREATE SCHEMA foo;
+  SELECT msar.get_schema_oid('foo') INTO sch_oid;
 
-CREATE OR REPLACE FUNCTION test_rename_schema() RETURNS SETOF TEXT AS $$
-BEGIN
-  PERFORM __setup_alter_schema();
-  PERFORM msar.rename_schema(
-    old_sch_name => 'alter_me',
-    new_sch_name => 'altered'
-  );
-  RETURN NEXT hasnt_schema('alter_me');
+  PERFORM msar.patch_schema('foo', '{"name": "altered"}');
+  RETURN NEXT hasnt_schema('foo');
   RETURN NEXT has_schema('altered');
-END;
-$$ LANGUAGE plpgsql;
+  RETURN NEXT is(obj_description(sch_oid), NULL);
+  RETURN NEXT is(msar.get_schema_name(sch_oid), 'altered');
 
+  PERFORM msar.patch_schema(sch_oid, '{"description": "yay"}');
+  RETURN NEXT is(obj_description(sch_oid), 'yay');
 
-CREATE OR REPLACE FUNCTION test_rename_schema_using_oid() RETURNS SETOF TEXT AS $$
-BEGIN
-  PERFORM __setup_alter_schema();
-  PERFORM msar.rename_schema(
-    sch_id => 'alter_me'::regnamespace::oid,
-    new_sch_name => 'altered'
-  );
-  RETURN NEXT hasnt_schema('alter_me');
-  RETURN NEXT has_schema('altered');
-END;
-$$ LANGUAGE plpgsql;
+  -- Edge case: setting the description to null doesn't actually remove it. This behavior is
```

Comment:
I think it's fine. At some point, we should put some effort into making this behavior consistent across the code base.